lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -55,7 +55,7 @@
       window.ace_editor = editor;
 
       editor.setTheme("ace/theme/tomorrow");
-      editor.setKeyboardHandler("ace/keyboard/vim");
+      editor.setKeyboardHandler("ace/keyboard/emacs");
```

Comment:
I change the keyboard to emacs to make it more comfortable for non-vim users, since it does not involve something like 'insert/command/visual mode'. I have mistakenly hard-coded my preference (vim) in the code and the commit is to revert it.